test/tf_exec.cpp

```diff
@@ -110,13 +110,23 @@ TEST(tf_exec, axpy) {
 
 void AssertTensorEquals(tf::Tensor T1, tf::Tensor T2) {
   auto T_size = T1.flat<float>().size();
+  auto T1_data = T1.flat<float>().data();
+  auto T2_data = T2.flat<float>().data();
   for (int k = 0; k < T_size; k++) {
-    auto a = T1.flat<float>().data()[k];
-    auto b = T2.flat<float>().data()[k];
+    auto a = T1_data[k];
+    auto b = T2_data[k];
     EXPECT_FLOAT_EQ(a, b);
   }
 }
 
+void AssignInputValues(tf::Tensor& A) {
+  auto A_flat = A.flat<float>();
```

Comment:
Variable names use snake_case. Please review this document about the coding standards: https://docs.google.com/document/d/1Xi0rIkMeAmV9ofuOkkmaRoaWFv0GzYkaJMiNq5LtVHo/edit#heading=h.jvtff2kbxiny
Even though you merely followed style in the existing code (which was wrong to begin with) - please update this function following the coding convention.